web/skins/classic/includes/functions.php

```diff
@@ -1021,7 +1021,17 @@ function xhtmlFooter() {
   $skinJsFile = getSkinFile('js/skin.js');
 ?>
   <script src="<?php echo cache_bust($skinJsFile) ?>"></script>
-  <script nonce="<?php echo $cspNonce; ?>">$j('.chosen').chosen({disable_search_threshold: 10});</script>
+  <script nonce="<?php echo $cspNonce; ?>">
```

Comment:
Instead of having this in php, let's stick the function in skin.js

Reply:
No problem. I thought about it, but decided not to touch it for now.
I don't know what to do with the $cspNonce token yet, but I'll look at the code

Reply:
remove it. Remove the entire script section.

Reply:
I transferred script
The $cspNonce token now applies to entire main JS script.
I have no problems with tests.
I haven’t tested the multi-server mode, but in theory there shouldn’t be any problems.